packages/cli/src/commands/labs/index.ts

```diff
@@ -0,0 +1,87 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import * as Heroku from '@heroku-cli/schema'
+import {Args, ux} from '@oclif/core'
+import {sortBy} from 'lodash'
+
+interface Features {
+  currentUser: any,
+  user: any,
+  app?: any,
+}
+
+function printJSON(features: Heroku.Account | Heroku.AccountFeature | Heroku.AppFeature) {
+  ux.log(JSON.stringify(features, null, 2))
+}
+
+function printFeatures(features: any) {
```

Comment:
```suggestion
function printFeatures(features: {name: string, enabled: boolean, description: string} ): void {
```

Reply:
@justinwilaby  Oof, was still running into some type errors after trying this on for size. Maybe we could pair post merge or pair next week on it?